frameworks/Crystal/amber/src/views/fortune/index.ecr

```diff
@@ -1,12 +1 @@
-<table>
-  <tr>
-  <th>id</th>
-  <th>message</th>
-  </tr>
-  <% fortunes.each do |fortune| %>
-    <tr>
-    <td><%= fortune.id %></td>
-    <td><%= HTML.escape(fortune.message || "") %></td>
-    </tr>
-  <% end %>
-</table>
\ No newline at end of file
+<table><tr><th>id</th><th>message</th></tr><% fortunes.each do |fortune| %><tr><td><%= fortune.id %></td><td><%= HTML.escape(fortune.message || "") %></td></tr><% end %></table>
```

Comment:
@faustinoaq this doesn't seem like a realistic template.  I would rather keep the spaces so if someone comes and looks at this Amber project, they can see what is going on.  

Do you anticipate a significant increase in performance by removing the spaces?

Reply:
Oh, I see let me revert this changes

Reply:
Done!